pmultiqc/modules/maxquant/maxquant.py

```
@@ -588,3 +621,15 @@ def draw_report_plots(self):
         }
 
         add_group_modules(self.section_group_dict, "")
+
+    def _safe_draw(self, draw_func, *args, error_name=None):
+        """Safely execute a drawing function with error handling."""
+        try:
+            draw_func(*args)
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)